fixtures/vuejs-css-modules/main_v3.js

```diff
@@ -0,0 +1,4 @@
+import { createApp } from 'vue';
+import App from './App'
+
+createApp(App).mount('#app');
```

Comment:
vue2 code is pretty much compat with vue3, except for how you originally render the app. So I have v2 and v3 entry files.